docs/iris/src/whatsnew/1.13.rst

```diff
@@ -0,0 +1,37 @@
+What's New in Iris 1.13
+***********************
+
+:Release: 1.13
+:Date: 2017-05-17
+
+
+This document explains the new/changed features of Iris in version 1.13
+(:doc:`View all changes <index>`.)
+
+Iris 1.13 Features
+==================
+
+* Allow the reading of NAME trajectories stored by time instead of by particle number.
+* An experimental link to python-stratify via :mod:`iris.experimental.stratify`.
+* Data arrays may be shared between cubes, and subsets of cubes, by using the :meth:`iris.cube.share_data` flag.
+
+
+Bug Fixes
+=========
+
+* The bounds are now set correctly on the longitude coordinate if a zonal mean diagnostic has been loaded from a PP file as per the CF Standard.
+* NetCDF loading will now determine whether there is a string-valued scalar label, i.e. a character variable that only has one dimension (the length of the string), and interpret this correctly.
+* A line plot of geographic coordinates (e.g. drawing a trajectory) wraps around the edge of the map cleanly, rather than plotting a segment straight across the map.
+* Preserve lazy data in PP save-pairs generation, so you can work with a list of save cubes and/or fields without excessive memory requirements (as intended for saving).
+* When two cubes have coordinates which do not match, an error should be raised if a user tries to perform arithmetic operations on them. These cases are now caught by the multiply and divide operators as well as the add and subtract operators, which already worked correctly.
```

Comment:
Or maybe 'An error should be raised if a user......'

The fact that it is now caught by divide and multiply is explained later.